docs/docs/30-how-to-guides/20-managing-credentials.md

```diff
@@ -106,6 +106,41 @@ Refer to
 [the advanced section of the installation guide](./10-installing-kargo.md#advanced-installation)
 for more details.
 
+While doing this you must ensure that the Kargo controller has the appropriate
+permissions to read `Secret` resources in these namespaces. This is necessary because,
+as of Kargo v1.0.0, the controller does not have cluster-wide access to `Secret`s. Permissions
+are granted dynamically on a per-Project basis as `Projects` are created.
```

Comment:
I have new thoughts about how to automate this. I will open a separate issue.

These changes are still worthwhile because this is the current state of things.